testcode/assembler/Makefile

```diff
@@ -1,8 +1,13 @@
 
```

Comment:
Have you considered putting your test into https://github.com/cc65/cc65/tree/master/test/misc instead? Then it would become part of the automatic regression testing...